crates/ln-dlc-node/src/tests/just_in_time_channel/fail_intercepted_htlc.rs

```diff
@@ -93,12 +102,19 @@ async fn fail_intercepted_htlc_if_connection_lost_after_funding_tx_generated() {
     let invoice_amount = 10_000;
     setup_coordinator_payer_channel(invoice_amount, &coordinator, &payer).await;
 
+    let liquidity_request = LiquidityRequest {
```

Comment:
Given we use the same values all over the place you could add a helper function creating a default only for tests.